pom.xml

```diff
@@ -93,6 +93,11 @@
         <artifactId>maven-deploy-plugin</artifactId>
         <version>2.8.2</version>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>3.0.0-M1</version>
```

Comment:
@almerico I changed this when resolving conflicts in the rebase - does it need to be `2.18.1` ?

Reply:
Last time I tried newer versions didn’t work in pipelines

Reply:
OK I guess we need to use that version then